src/nl/hannahsten/texifyidea/action/ForwardSearchAction.kt

```diff
@@ -22,11 +20,15 @@ open class ForwardSearchAction(var viewer: PdfViewer? = null) : EditorAction(
 
         val document = textEditor.editor.document
         val line = document.getLineNumber(textEditor.editor.caretModel.offset) + 1
-
-        when (viewer) {
-            is ExternalPdfViewer -> (viewer as ExternalPdfViewer).forwardSearch(null, file.path, line, project, focusAllowed = true)
-            is InternalPdfViewer -> (viewer as InternalPdfViewer).conversation?.forwardSearch(null, file.path, line, project, focusAllowed = true)
-            else -> return
+        try {
+            viewer?.forwardSearch(null, file.path, line, project, focusAllowed = true)
+        }
+        catch (e: Exception) {
+//            Notification(
+//                "LaTeX", "Forward search error", "Could not execute forward search with ${viewer?.displayName}.",
+//                NotificationType.WARNING
+//            ).notify(project)
+            // Ignore exception, such as when the file is not found.
```

Comment:
I don't think we should be ignoring all exceptions here. Maybe we can convert certain 'expected' exceptions to a notification, but even a file not found should not happen I think?